app/common/services/endpoints/user-endpoint.js

```diff
@@ -2,11 +2,23 @@ module.exports = [
     '$resource',
     '$rootScope',
     'Util',
+    'CacheFactory',
 function (
     $resource,
     $rootScope,
-    Util
+    Util,
+    CacheFactory
 ) {
+    var cache;
+
+    if (!(cache = CacheFactory.get('userCache'))) {
```

Comment:
I'll check this but I added it because the spec test was failing as the cache had already been created. I'm not sure why that failure would occur but it doesn't seem that the lib checks for existence itself. So I should propagate this to all the endpoints.